deployments/puppet/README.md

```diff
@@ -14,6 +14,7 @@ class accepts the following parameters:
     ```ruby
     $config = {
       signalFxAccessToken: "MY_TOKEN",
+      signalFxRealm: "us1",
```

Comment:
Why defaulting to us1? 

Also if they specify signalFxRealm they won't have to specify ingesturl and apiurl right?

Reply:
That's not a default, just a documented example.

Reply:
Yes, if they specify realm the other options are automatically derived by the agent.